### PR TITLE
release: released 1.5.0 version.

### PR DESCRIPTION
**Pull request by shuaijinchao**
### What this PR does / why we need it:
released 1.5.0 version.
@apache/apisix-committers please do have a check.
@membphis After PR merge, please upload `apisix-1.5-0.rockspec` to https://luarocks.org/modules/apisix/apisix

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: doc/zh-cn/how-to-build.md [hunk 1]
```diff
@@ -34,20 +34,20 @@ Apache APISIX 的运行环境需要 Nginx 和 etcd，
...
-cd apache-apisix-1.4.1
+cd apache-apisix-1.5
 make deps
 ```
 
 ### 通过 RPM 包安装（CentOS 7）
 
 ```shell
-sudo yum install -y https://github.com/apache/incubator-apisix/releases/download/1.4.1/apisix-1.4.1-0.el7.noarch.rpm
+sudo yum install -y https://github.com/apache/incubator-apisix/releases/download/1.5/apisix-1.5-0.el7.noarch.rpm
```
Comment (membphis) [inline doc/zh-cn/how-to-build.md:50]: update from `https://github.com/apache/incubator-apisix` to `https://github.com/apache/apisix`

Comment (membphis) [reply doc/zh-cn/how-to-build.md:50]: please fix the other points

**Comment by membphis** [comment]
@shuaijinchao please rebase your branch.

image

**Comment by shuaijinchao** [comment]
many thx @membphis for helping me solve this problem.

**Pull request merged**